glean_parser/templates/kotlin.jinja2

```diff
@@ -19,7 +19,7 @@ Jinja2 template is not. Please file bugs! #}
         )
 {% if lazy %}    }{% endif %}{% endmacro %}
 
-@file:Suppress("PackageNaming")
+@file:Suppress("PackageNaming", "MaxLineLength")
```

Comment:
This is triggered when a labeled metric has lots of labels and the `listOf(...)` construction is really long.  Fixing that properly would require a tool that can properly re-indent Kotlin code.  `ktlint` can do that, but that would make `ktlint` a hard requirement of glean_parser, which is currently isn't.  (ktlint is only used for unit tests right now).

Reply:
Good catch, no need to depend on ktlint for this..